modules/minimint-wallet/src/bitcoind.rs

```diff
@@ -1,13 +1,11 @@
-use std::error::Error;
-
 use crate::Feerate;
 use async_trait::async_trait;
 use bitcoin::{BlockHash, Transaction};
 use thiserror::Error;
 
 #[derive(Debug, Error)]
 #[error(transparent)]
-pub struct BitcoinRpcError(#[from] pub Box<dyn Error + Sync + Send>);
+pub struct BitcoinRpcError(#[from] pub anyhow::Error);
```

Comment:
Same here, doesn't need to be a newtype imo, although I'm not sure about what the idiomatic thing would be to do. The existence of `#[error(transparent)]` indicates that newtype errors aren't all that uncommon.

Reply:
> 

"... This would be appropriate for enums that need an “anything else” variant." (from thiserror docs) I think in this case it shouldn't be a newtype

Reply:
I'm undecided on the newtype here. Seem mechanically kind of unnecessary, but on the other hand - it does convey a specific meaning in case stack trace (chain cause) is being printed etc.

Could `BitcoinRpcError` be an `enum` with [bitcoincore_rpc](https://docs.rs/bitcoincore-rpc/latest/bitcoincore_rpc/index.html)::[Error](https://docs.rs/bitcoincore-rpc/latest/bitcoincore_rpc/enum.Error.html#)  + `Other(anyhow::Error)`?